app/assets/javascripts/admin/addon/templates/flags-index.hbs

```diff
@@ -0,0 +1 @@
+<AdminFlags />
```

Comment:
Love to see it. I hate going back to old routes and seeing 100000 things in the route template :grimacing: